src/Calendar/__stories__/Calendar.stories.js

```diff
@@ -70,6 +73,11 @@ export const rangeSelection = () => (
     <Calendar enableRangeSelection />
 );
 
+export const weekdayStart = () => {
```

Comment:
So I kept a story to showcase a different starting week in the docs but added the prop to the dev story as well.